controllers/cluster_controller.go

```diff
@@ -199,6 +142,99 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *capi.Cluster
 	return requeue(), nil
 }
 
+func deleteVintageCluster(ctx context.Context, log logr.Logger, client ctrlclient.Client, cluster *capi.Cluster) error {
+	log.Info(fmt.Sprintf("Cluster %s/%s is being deleted", cluster.Namespace, cluster.Name))
```

Comment:
_Out of scope for these PRs_, but I wanted to mention it for the future: `log.Info(fmt.Sprintf(...))` is not a good way of logging. Usually we can set values like `log = log.WithValues("cluster", clusterName)` and then logging should be without any %placeholders such as `log.Info("Deleting cluster")` since the structured fields would then be added by the logging framework in a parseable format.

Reply:
Done, addressed. I've only left for Apps.
```
	log.Info(fmt.Sprintf("App %s/%s is being deleted", app.Name, app.Namespace))
```